src/Grpc/JsonTranscoding/src/Microsoft.AspNetCore.Grpc.Swagger/Internal/GrpcJsonTranscodingDescriptionProvider.cs

```diff
@@ -182,6 +182,11 @@ private static string ResolvePath(HttpRoutePattern httpRoutePattern, Dictionary<
                 sb.Append(httpRoutePattern.Segments[i]);
             }
         }
+        if (httpRoutePattern.Verb != null)
```

Comment:
nit:
```suggestion
        if (!string.IsNullOrEmpty(httpRoutePattern.Verb))
```

Reply:
Purposefully using `!= null` for trailing colon. (i.e. empty verb)